CHANGELOG.md

```diff
@@ -37,6 +37,7 @@ All notable, unreleased changes to this project will be documented in this file.
   - Event sent when CSV export for gift cards is completed.
 - Add `PRODUCT_EXPORT_COMPLETED` webhook - #13787, by @Air-t
   - Event sent when CSV export for products is completed.
+
```

Comment:
The changelog entry "Dry runs enabled:" below should be removed right?

Reply:
not sure why it was not removed by revert, but yes the whole record should be removed